config/initializers/devise.rb

```diff
@@ -216,14 +216,14 @@
   # :time  = Re-enables login after a certain amount of time (see :unlock_in below)
   # :both  = Enables both strategies
   # :none  = No unlock strategy. You should handle unlocking by yourself.
-  # config.unlock_strategy = :both
+  config.unlock_strategy = :both
```

Comment:
This together with `config.unlock_in = 1.hour` means the account will automatically unlock after 1 hour. We can change that if people feel like it, but since our main motivation for this was deterring brute-force attacks it seemed like an ok setting.